io.gitlab.liferooter.TextPieces.gnome.yaml

```diff
@@ -0,0 +1,47 @@
+# SPDX-FileCopyrightText: 2023 Gleb Smirnov <glebsmirnov0708@gmail.com>
+#
+# SPDX-License-Identifier: CC0-1.0
+
+id: io.gitlab.liferooter.TextPieces.gnome
+
+runtime: org.gnome.Platform
+runtime-version: '46'
+
+sdk: org.gnome.Sdk
+sdk-extensions:
+- org.freedesktop.Sdk.Extension.rust-stable
+
+command: textpieces-gnome
+
+finish-args:
+- --socket=fallback-x11
+- --socket=wayland
+- --device=dri
+- --env=RUST_LOG=textpieces_gnome=debug
+- --env=G_MESSAGES_DEBUG=none
+- --env=RUST_BACKTRACE=full
+- --talk-name=org.freedesktop.Flatpak
+- --filesystem=~/.local/share/textpieces
+- --filesystem=~/.var/app/com.github.liferooter.textpieces
+
+modules:
+
+- name: blueprint-compiler
+  buildsystem: meson
+  cleanup:
+  - '*'
+  sources:
+  - type: git
+    url: https://gitlab.gnome.org/jwestman/blueprint-compiler
+    tag: v0.12.0
+
+- name: textpieces-gnome
+  buildsystem: meson
+  build-options:
+    append-path: /usr/lib/sdk/rust-stable/bin
+    build-args:
+    - --share=network
```

Comment:
build MUST be offline. this hasn't changed.

Reply:
How can I build Rust application with dependencies from crates.io offline?

Reply:
Just has read the docs more carefully. Will fix.